lab_1/main.py

```diff
@@ -69,9 +128,34 @@ def get_concordance(tokens: list, word: str, left_context_size: int, right_conte
     right_context_size = 3
     --> [['man', 'is', 'happy', 'the', 'dog', 'is'], ['dog', 'is', 'happy', 'but', 'the', 'cat']]
     """
-    pass
+    concordance = []
+    left_context = []
+    right_context = []
+    if type(tokens) == list and type(word) == str and type(left_context_size) == int and type(right_context_size) == int:
+        if left_context_size <= 0 and right_context_size > 0\
+                or left_context_size > 0 and right_context_size <= 0\
+                or left_context_size > 0 and right_context_size > 0:
+            operative_tokens = tokens[:]
+            word = word.lower()
+            while word in operative_tokens:
```

Comment:
Do you really need to use `while` here? Please, explain.

Reply:
That's all I could think of! At the end of the loop, the first "word" in the list is removed, and until each use of this word is removed, a list of tokens is added to the concordance...
I really don't know how to else to do it...